corehq/apps/app_manager/static/app_manager/js/releases/app_view_release_manager.js

```diff
@@ -18,8 +18,34 @@ hqDefine("app_manager/js/releases/app_view_release_manager", function () {
         var releasesMain = releasesMainModel(o);
         el.koApplyBindings(releasesMain);
         _.defer(function () { releasesMain.goToPage(1); });
-    }
 
+        var releaseControlEl = $('#release-control');
+        if (releaseControlEl.length) {
```

Comment:
Hey @orangejenny 
will let @akashkj any details I missed.

From the design perspective, the toggle looked better to be above the releases table and hence the way javascript happened. We did try it next to "Show only released versions" first but that threw other elements off and having toggles icons/buttons next to a text toggle to show released versions wasn't looking pleasing as well.
It made sense to keep it on top where its visible.

Feel free to play around with HTML, quick link to an app [here](https://staging.commcarehq.org/a/fhir/apps/view/d03d683c180149c4b5f54b8ba6ebcfca/). Things did become harder when we got that toggle outside the table, so if we can move it back, it would be less code only.